test_programs/execution_success/global_slice_rc_regression_8259/src/main.nr

```diff
@@ -0,0 +1,9 @@
+global G_C: [bool] = &[true, false, true];
+fn main(a: bool) {
+    println(func_1(a, G_C));
```

Comment:
We cannot return slices from main so this was the simplest way to trigger the same bug but with slices.